src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/client/model/prisoner/Identifier.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.hmpps.personrecord.client.model.prisoner
```

Comment:
Just wondering why these are in a prisoner package, are they not used for all people?

Reply:
this format is specific to prisoner-search API as far as I know

Reply:
https://prisoner-search-dev.prison.service.justice.gov.uk/swagger-ui/index.html?configUrl=/v3/api-docs/swagger-config#/Popular/findByPrisonNumber